tests/integration-tests/tests/test_awsbatch.py

```diff
@@ -10,24 +10,69 @@
 # This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
 # See the License for the specific language governing permissions and limitations under the License.
 import logging
+import re
 
 import pytest
+from retrying import retry
 
+from assertpy import assert_that
 from remote_command_executor import RemoteCommandExecutor
+from time_utils import minutes, seconds
 
 
 @pytest.mark.regions(["us-east-1", "eu-west-1", "cn-north-1", "us-gov-west-1"])
 @pytest.mark.instances(["c5.xlarge", "t2.large"])
 @pytest.mark.dimensions("*", "*", "alinux", "awsbatch")
-def test_job_submission(region, os, instance, scheduler, pcluster_config_reader, clusters_factory, test_datadir):
+def test_simple_job_submission(region, os, instance, scheduler, pcluster_config_reader, clusters_factory, test_datadir):
     cluster_config = pcluster_config_reader()
     cluster = clusters_factory(cluster_config)
     remote_command_executor = RemoteCommandExecutor(cluster)
-    result = remote_command_executor.run_remote_command("env")
-    logging.info(result.stdout)
-    result = remote_command_executor.run_remote_command(["echo", "test"])
-    logging.info(result.stdout)
-    result = remote_command_executor.run_remote_script(
-        str(test_datadir / "test_script.sh"), args=["1", "2"], additional_files=[str(test_datadir / "data_file")]
+
+    logging.info("Testing inline submission.")
+    _test_job_submission(remote_command_executor, "awsbsub --vcpus 2 --memory 256 --timeout 60 sleep 1")
+
+    # https://issues.amazon.com/issues/CFNCLUSTER-557
```

Comment:
I think we should remove this internal link

Reply:
Indeed! Removed, thanks!